src/Sulu/Bundle/MediaBundle/Resources/js/stores/SingleMediaSelectionStore/SingleMediaSelectionStore.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import type {IObservableValue} from 'mobx';
+import {action, computed, observable} from 'mobx';
+import {ResourceRequester} from 'sulu-admin-bundle/services';
+import type {MediaItem} from '../MultiMediaSelectionStore/types';
```

Comment:
That's a bit a strange dependency... Wouldn't it be a better idea to use the `Media` type from the `MediaBundle/Resources/js/types.js` file in both places?